functional_tests/features/bundle_page.feature

```diff
@@ -0,0 +1,18 @@
+Feature: Publishing Officer can draft, edit, and publish bundles
+
+
+    Scenario: A Publishing Officer can add Bundles with Publishing Officer as the creator.  The bundle can be seen and accessed.
```

Comment:
This should be split into separate scenarios:
* A Publishing Officer can add Bundles with Publishing Officer as the creator
* The bundle can be seen and accessed
* Once the creator is removed from the bundle then the bundle can still be seen and accessed

If you really want you could probably combine the first two (e.g. we can confirm that the bundle was successfully created by viewing it), but the last one should definitely be a separate test.